pysat/tests/test_constellation.py

```diff
@@ -16,7 +16,8 @@ def setup(self):
         """Create instruments and a constellation for each test
         """
         self.instruments = [pysat.Instrument('pysat', 'testing',
-                                             clean_level='clean')
+                                             clean_level='clean',
+                                             update_files=True)
```

Comment:
We may need to set the temporary_file_list flag here. @jklenzing I believe I saw you had modified some testing configurations to prevent changes to a user setup. I was having problems with testing at one point and thus scattered `update_files=True` around. I may have reintroduced stuff you removed.